src/nl/rubensten/texifyidea/TexifyIcons.java

```diff
@@ -401,6 +401,11 @@ public static Icon getIconFromExtension(String extension) {
                 return PDF_FILE;
             case "synctex.gz":
                 return SYNCTEX_FILE;
+            case "png":
```

Comment:
Why did you make these changes, what value are they adding? I mean, I don't think adding a todo does anything useful. What would maybe be helpful, is commenting on #820 to ask whether we want an icon for these or not (I'm not sure).

Reply:
This commit does not make any sense in this branch :D it has not to do anything with the purpose of this PR, I don't know why I commited it here :D
Is there an possibility to revert/undo this commit without making the history much bigger?

Reply:
Ah okay. Yes, use git revert or IJ (probably right-click on commit or something).

Reply:
Okay, I knew about git revert, but I hoped there could be a way to remove this commit and do not add an other one. But when this is the correct way, than we do it this way.

Reply:
@Stainzl There is (using hard reset and force push), but this is highly undesirable in general and also in this case as I said. One commit extra is no problem at all. I hope you noticed I already reverted it :)

Reply:
Also a tip in general: first pull a brach before you continue working on it. Would have avoided an empty merge commit this case ;)

Reply:
Okay, yes, a force push is not the best tool to use :D
Which merge commit do you mean? I first pulled your change and then merged the master. Or am I missing a commit?

Reply:
I meant 26c3e1bf51d10882cfbdfa53b8b240962fd5a1c0, but it is a little bit more tricky to understand why this is empty. I think it's best not to worry about that now and leave it as it is :)

Reply:
Okay, what do you mean with "empty", I think I do not completly understand what you are talking about, because in this commit is a big list of diffrent files?
I merged it because github told me on this page I should do and I wanted to verify that there aren't any merge conflicts which may break anything.

Reply:
No, I was confused because GitHub apparently only shows the diff after you click on 'show diff', this only happens for me when I click on the link to the commit below, not when I go to that same link after putting it in this message.
I'm not sure why GitHub would ask you that though, but as I said doesn't matter.

Reply:
GitHub asks because I want to merge to master and GitHub noticed that the branch I want to merge does not contain all commits from master, so it is possible to get merge conflicts when you try to merge my branch to master. And to prevent this, GitHub tells you to merge master to your brnach first, because then a fast forward merge is always possible